tests/kernel/run_ltp.pm

```diff
@@ -351,6 +342,11 @@ sub run_post_fail {
     my ($self, $msg) = @_;
 
     $self->fail_if_running();
+
+    if ($self->{force_softfail}) {
+        $self->{result} = 'softfail';
+    }
```

Comment:
`force_soft_failure` from https://github.com/os-autoinst/os-autoinst/blob/master/testapi.pm#L220 looks like a cleaner way than forcing it here but this works as well of course.

Reply:
`force_soft_failure()` would add a new test output box to the results. I don't want to add another output box, I just want to undo the status change to `fail` which was forced by os-autoinst after `run_ltp::run()` raised an exception.